tests/sqlite3/Makefile.basic

```diff
@@ -0,0 +1,12 @@
+COMPILER = g++
```

Comment:
Is it really a good idea to maintain and add these hand-written makefiles? I'm not a big fan of CMake myself, but IMO it's really better to have one, single build system rather than two in parallel...